src/logging.cpp

```diff
@@ -86,6 +92,32 @@ namespace logging {
 #endif
     };
 
+#ifdef __ANDROID__
+    android_LogPriority android_priority;
```

Comment:
```suggestion
    android_LogPriority android_priority{};
```
If this `android_LogPriority` is a built-in type, then it should be initialized to some default value at least. If it's a class/struct you can leave it as it is.